### PR TITLE
index.html: Corrected grammatical errors

### PR DESCRIPTION
**Pull request by stellargo**
A few grammatical and syntactical errors were fixed in the sponsors section.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2449,10 +2449,10 @@ <h3 class = "wow fadeInDown text-center">Keynote Speakers</h3>
             <h3 class="wow fadeInDown">Sponsors</h3>
           </div>
           <div class="tg-description">
-            <p>PyCon India is entirely driven by volunteer. Sponsoring the event helps to sustain, grow the conference along with the community. Sponsors help to make the conference affordable for community, and maintaining the inventory for the conference.</p>
+            <p>PyCon India is entirely driven by volunteers. Sponsoring the event helps to sustain and grow the conference as well as the community. Sponsors help in making the conference affordable, and maintaining the inventory for the conference.</p>
```
Comment (pythonhacker) [inline index.html:2452]: This change is acceptable.

File: index.html [hunk 2]
```diff
@@ -2449,10 +2449,10 @@ <h3 class = "wow fadeInDown text-center">Keynote Speakers</h3>
...
             <br>
             <br>
-            <p style="font-size: x-large;">Looking to become a sponsor?</p>
+            <p style="font-size: x-large;">Looking forward to becoming a sponsor?</p>
```
Comment (pythonhacker) [inline index.html]: This change is not acceptable.

**Comment by pythonhacker** [comment]
Thanks. 

But not all of these are acceptable.

Example: "Looking forward to becoming a sponsor" - No it is not "looking forward". It is "Looking to become a sponsor" itself. Looking forward has intonations of longing and cant-wait. This is clearly not the case here. 

Since this modifies the text of main page and changes the meaning, it has to be better than this. See my inline comments in the commit.

File: templates/sections/_sponsors.html [hunk 3]
```diff
@@ -11,10 +11,10 @@
             <h3 class="wow fadeInDown">Sponsors</h3>
           </div>
           <div class="tg-description">
-            <p>PyCon India is entirely driven by volunteer. Sponsoring the event helps to sustain, grow the conference along with the community. Sponsors help to make the conference affordable for community, and maintaining the inventory for the conference.</p>
+            <p>PyCon India is entirely driven by volunteers. Sponsoring the event helps to sustain and grow the conference as well as the community. Sponsors help in making the conference affordable, and maintaining the inventory for the conference.</p>
             <br>
             <br>
-            <p style="font-size: x-large;">Looking to become a sponsor?</p>
+            <p style="font-size: x-large;">Looking forward to becoming a sponsor?</p>
```
Comment (pythonhacker) [inline templates/sections/_sponsors.html]: Same here - not acceptable.

File: templates/sections/_sponsors.html [hunk 4]
```diff
@@ -11,10 +11,10 @@
             <h3 class="wow fadeInDown">Sponsors</h3>
           </div>
           <div class="tg-description">
-            <p>PyCon India is entirely driven by volunteer. Sponsoring the event helps to sustain, grow the conference along with the community. Sponsors help to make the conference affordable for community, and maintaining the inventory for the conference.</p>
+            <p>PyCon India is entirely driven by volunteers. Sponsoring the event helps to sustain and grow the conference as well as the community. Sponsors help in making the conference affordable, and maintaining the inventory for the conference.</p>
```
Comment (pythonhacker) [inline templates/sections/_sponsors.html:14]: Fine!

**Comment by stellargo** [comment]
@pythonhacker Ok I get your point. In that case 'Want to become a sponsor?" might be better according to me. If you feel that is better too let me know. As of now I have reverted it to 'Looking to become a sponsor?'

**Pull request merged**